### PR TITLE
Add SQL Trace Files to Repository

### PR DESCRIPTION
**Pull request by dniu16**
Move SQL trace files from noise-testfiles repository to terrier
Adjust run_junit.py to accommodate the change
Enable trace files to include comments for GenerateTrace
Contains the latest code

### CODE REVIEW

File: script/testing/junit/build.xml [hunk 1]
```diff
@@ -31,7 +31,6 @@
   <path refid="lib.classpath"/>
   <pathelement location="${out.dir}" />
 </path>
-
```
Comment (apavlo) [inline script/testing/junit/build.xml:34]: Why do you keep removing these newlines?

File: script/testing/junit/run_junit.py [hunk 2]
```diff
@@ -70,20 +48,15 @@ def download_git_repo():
...
-        
+
     # Step 2: Run the trace test for each file that we find
     # Each directory represents another set of SQL traces to test.
-    noise_trace_dir = os.path.join(os.getcwd(), constants.TESTFILES_REPO, constants.TESTFILES_REPO_TRACE_DIR)
-    for test_type in os.listdir(noise_trace_dir):
-        type_dir = os.path.join(noise_trace_dir, test_type)
-        if not os.path.isdir(type_dir): continue
-
+    noise_trace_dir = os.path.join(base_path, constants.JUNIT_DIR, constants.REPO_TRACE_DIR)
```
Comment (apavlo) [inline script/testing/junit/run_junit.py:54]: Is `constants.JUNIT_DIR` redundant here?

File: script/testing/junit/src/GenerateTrace.java [hunk 3]
```diff
@@ -19,13 +20,17 @@
...
 public class GenerateTrace {
     private static final String STATEMENT_OK = "statement ok";
+    private static final String STATEMENT_ERROR = "statement error";
     public static final String QUERY_I_NOSORT = "query I nosort";
     public static final String SEPARATION = "----";
+    public static final String DEST_DIR = "traces";
+    public static final String DEST_NAME = "output.test";
 
-    public static void main(String[] args) throws Throwable {
+    public static void main(String[] args) throws IOException, SQLException {
```
Comment (apavlo) [inline script/testing/junit/src/GenerateTrace.java:29]: Why did you change it to not be `Throwable`?

File: script/testing/junit/src/TracefileTest.java [hunk 4]
```diff
@@ -73,6 +72,7 @@ public void setUp() throws FileNotFoundException, SQLException {
                     dTest.add(cur);
                 }
                 catch (Throwable e) {
+                    System.out.println(mog.sql);
```
Comment (apavlo) [inline script/testing/junit/src/TracefileTest.java:75]: Is this print statement superfluous?

File: script/testing/junit/src/moglib/MogSqlite.java [hunk 5]
```diff
@@ -307,7 +307,8 @@ private void readRecordStatement(String line) throws IOException {
             this.br.mark(RECORD_READAHEAD_LIMIT);
             line = this.br.readLine();
 
-            if (null == line || line.startsWith("query") || line.startsWith("statement ok") || line.startsWith("statement error")) {
+            if (null == line || line.startsWith("query") || line.startsWith("statement ok")
+                    || line.startsWith("statement error") ||line.startsWith("#")) {
```
Comment (apavlo) [inline script/testing/junit/src/moglib/MogSqlite.java:311]: You should probably make these be constants.

File: script/testing/junit/src/TracefileTest.java (hunk 4 above)
Comment (dniu16) [reply script/testing/junit/src/TracefileTest.java:75]: Sorry it isn't supposed to be there. It was added back to the code when I rebased and was never run...
